src/views/hash.clj

```diff
@@ -0,0 +1,17 @@
+(ns views.hash
+  (:require
+    [hasch.benc :refer [magics PHashCoercion -coerce digest coerce-seq xor-hashes encode-safe]]
+    [hasch.core :refer [edn-hash]]
+    [hasch.platform :refer [encode md5-message-digest]]))
+
+(extend-protocol PHashCoercion
+  java.math.BigDecimal
+  (-coerce [this md-create-fn write-handlers]
+    (encode (:number magics) (.getBytes (.toString this) "UTF-8"))))
+
+(defn hash
```

Comment:
Do we not need to exclude the core hash function using (:refer-clojure :exclude [hash])?